Formula/kde-kguiaddons.rb

```diff
@@ -0,0 +1,66 @@
+class KdeKguiaddons < Formula
+  desc "Addons to QtGui"
+  homepage "https://api.kde.org/frameworks/kguiaddons/html/index.html"
+  url "https://download.kde.org/stable/frameworks/5.79/kguiaddons-5.79.0.tar.xz"
+  sha256 "bc00488e123a1f7905682393b140b80c8340cb131cc0baa3fb6ad575878f1c85"
+  license all_of: [
+    "LGPL-2.0-only",
+    "LGPL-2.0-or-later",
+    any_of: ["GPL-2.0-only", "GPL-3.0-only"],
+    any_of: ["LGPL-2.1-only", "LGPL-3.0-only"],
```

Comment:
```suggestion
    { any_of: ["GPL-2.0-only", "GPL-3.0-only"] },
    { any_of: ["LGPL-2.1-only", "LGPL-3.0-only"] },
```

Technically only the first (`GPL`) set needs to be enclosed in the `{}` but I think it looks better to have them both enclosed rather than just the one.

I assume that it needs to be `any_of: ["GPL-2.0-only", "GPL-3.0-only"]` and can't just be `GPL-2.0-or-later`, right?

Reply:
> I assume that it needs to be any_of: ["GPL-2.0-only", "GPL-3.0-only"] and can't just be GPL-2.0-or-later, right?

Indeed, this is for those files that are licenses with https://community.kde.org/Policies/Licensing_Policy#GPL_.28GPL-2.0-only_OR_GPL-3.0-only_OR_LicenseRef-KDE-Accepted-GPL.29 and https://community.kde.org/Policies/Licensing_Policy#LGPL_.28LGPL-2.1-only_OR_LGPL-3.0-only_OR_LicenseRef-KDE-Accepted-LGPL.29, respectively